blog/2020-02-angular9/README.md

```diff
@@ -112,10 +128,9 @@ ng generate interceptor
 Für Services wird ab Angular 6.0.0 standardmäßig die Option `providedIn: 'root'` verwendet (wir haben dazu im [Update-Artikel zu Angular 6](https://angular-buch.com/blog/2018-05-angular6) berichtet).
 Mit Angular 9 kommen neben `root` zwei neue Optionen für die Sichtbarkeit eines Providers hinzu: `any` und `platform`.
 
-* `root`: Die Anwendung erhält *eine einzige Instanz* des Services.
-* `any`: Jedes Modul der Anwendung erhält eine *eigene Instanz* des Services.
-* `platform`: Alle Anwendungen auf der Seite teilen sich *dieselbe Instanz*. Das ist vor allem im Kontext von [Angular Elements](https://angular.io/guide/elements) interessant, wenn mehrere Anwendungen auf einer Seite gebootstrappt werden.
-
+- `root`: Die Anwendung erhält _eine einzige Instanz_ des Services.
```

Comment:
warum?

Reply:
hmm das macht mein Formatter. Der sagt, dass `-` statt `*` genutzt werden soll. + formatOnSave in vscode.